### ISSUE TITLE
CIS Operator not working in OpenShift 4.11.x

### ISSUE THREAD

**Issue by wolfganghuse** [F5Networks/k8s-bigip-ctlr#2688]
#### Setup Details
 CIS Version : 2.X      
 Build: f5networks/k8s-bigip-ctlr:latest       
 BIGIP Version: Big IP 16.0.1 LTS     
 AS3 Version: 3.41      
 Agent Mode: AS3      
 Orchestration: OSCP      
 Orchestration Version:  4.11   
 Pool Mode: Cluster/Nodeport      
 Additional Setup details: 

#### Description  
Operator not available after installing on OpenShift 4.11
  

#### Steps To Reproduce
Select Operator from Marketplace and install with default settings

#### Expected Result 
Operator is able to create CIS Deployment

#### Actual Result  
Operator not starting successfully

#### Diagnostic Information
```{"level":"info","ts":1671795175.7291033,"logger":"cmd","msg":"Version","Go Version":"go1.18.4","GOOS":"linux","GOARCH":"amd64","helm-operator":"v1.22.0-ocp","commit":"aad1dd421c3ac4079caccfa048423189258bc6b9"}
{"level":"info","ts":1671795175.7296147,"logger":"cmd","msg":"Watch namespaces not configured by environment variable WATCH_NAMESPACE or file. Watching all namespaces.","Namespace":""}
I1223 11:32:56.780694       1 request.go:601] Waited for 1.03800421s due to client-side throttling, not priority and fairness, request: GET:https://172.30.0.1:443/apis/batch/v1beta1?timeout=32s
{"level":"info","ts":1671795178.3931117,"logger":"controller-runtime.metrics","msg":"Metrics server is starting to listen","addr":"127.0.0.1:8080"}
{"level":"info","ts":1671795178.3939831,"logger":"helm.controller","msg":"Watching resource","apiVersion":"cis.f5.com/v1","kind":"F5BigIpCtlr","namespace":"","reconcilePeriod":"1m0s"}
{"level":"info","ts":1671795178.3942533,"msg":"Starting server","path":"/metrics","kind":"metrics","addr":"127.0.0.1:8080"}
{"level":"info","ts":1671795178.3943439,"msg":"Starting server","kind":"health probe","addr":"[::]:8081"}
I1223 11:32:58.394359       1 leaderelection.go:248] attempting to acquire leader lease openshift-operators/f5-bigip-ctlr-operator...
E1223 11:32:58.404790       1 leaderelection.go:334] error initially creating leader election record: leases.coordination.k8s.io is forbidden: User "system:serviceaccount:openshift-operators:f5-bigip-ctlr-operator-controller-manager" cannot create resource "leases" in API group "coordination.k8s.io" in the namespace "openshift-operators"
E1223 11:33:01.243272       1 leaderelection.go:330] error retrieving resource lock openshift-operators/f5-bigip-ctlr-operator: leases.coordination.k8s.io "f5-bigip-ctlr-operator" is forbidden: User "system:serviceaccount:openshift-operators:f5-bigip-ctlr-operator-controller-manager" cannot get resource "leases" in API group "coordination.k8s.io" in the namespace "openshift-operators"
```

**Comment by trinaths** [comment]
based on error log, 

> E1223 11:32:58.404790       1 leaderelection.go:334] error initially creating leader election record: leases.coordination.k8s.io is forbidden: User "system:serviceaccount:openshift-operators:f5-bigip-ctlr-operator-controller-manager" cannot create resource "leases" in API group "coordination.k8s.io" in the namespace "openshift-operators"
> E1223 11:33:01.243272       1 leaderelection.go:330] error retrieving resource lock openshift-operators/f5-bigip-ctlr-operator: leases.coordination.k8s.io "f5-bigip-ctlr-operator" is forbidden: User "system:serviceaccount:openshift-operators:f5-bigip-ctlr-operator-controller-manager" cannot get resource "leases" in API group "coordination.k8s.io" in the namespace "openshift-operators"

There is an issue with the OCP cluster privileged access. Not a CIS/CIS-Operator issue.

**Comment by trinaths** [comment]
@wolfganghuse - We are able to reproduce this issue. Fix is to add “get” verb for resource leases.coordination.k8s.io in operator clusterRole. Fix will be published in 2.11.1 Operator release.
